packages/headless/src/features/triggers/triggers-state.ts

```diff
@@ -48,6 +48,15 @@ export interface TriggerState {
    * The notifications to display to the user after receiving notification triggers.
    */
   notifications: string[];
+
+  /**
+   * The modification that should be applied as a result of a query trigger.
+   */
+  queryModification: {
+    originalQuery: string;
+    modification: string;
```

Comment:
Instead of `modification`, I feel like `newQuery` would make more sense and would be more comparable to `originalQuery`.